cli/types/cypress.d.ts

```diff
@@ -3004,10 +3004,12 @@ declare namespace Cypress {
      */
     trashAssetsBeforeRuns: boolean
     /**
-     * The quality setting for the video compression, in Constant Rate Factor (CRF). The value can be false to disable compression or a value between 0 and 51, where a lower value results in better quality (at the expense of a higher file size).
+     * The quality setting for the video compression, in Constant Rate Factor (CRF). 
+     * The value can be false to disable compression or a value between 0 and 51, where a lower value results in better quality (at the expense of a higher file size).
+     * Optionally, a value of true can be provided to set a CRF of 32.
```

Comment:
```suggestion
     * The value can be false to disable compression.
     * The value of true can use the default CRF of 32.
     * A number between 0 and 51, where a lower value results in better quality (at the expense of a higher file size).
```

Also is false equivalent to `videoCompression=0`? or does `videoCompression=0` mean no compression at all?

Reply:
`false` is equivalent to `videoCompression=0` because `videoCompression=0` is essentially a lossless compress video which is the same as not compressing it.